app/assets/stylesheets/tag.scss

```diff
@@ -35,4 +35,15 @@ h1.tag {
       }
     }
   }
+  .span7 {
+    .pull-right {
+      &:first-child { max-width: 100%; }  
```

Comment:
Instead of this `first-child` "hack" I'd prefer adding another class to [this line](https://github.com/diaspora/diaspora/blob/develop/app/assets/templates/tag_following_action_tpl.jst.hbs#L1) (something like `tag-following-action`?) and instead of applying the style to `.pull-right:first-child` you could apply everything to `.tag-following-action`.